### DISCUSSION TITLE
本地编辑不通过的问题，出现报错

### DISCUSSION THREAD

**Discussion by 13237143205** [stone-zeng/fduthesis#322] [Q&A]
![image](https://github.com/stone-zeng/fduthesis/assets/109585419/2ad05123-b268-4a90-bd49-482a10a337bf)
textlive2023出现如上所示的报错，是什么原因呢？

**Comment by stone-zeng** [comment]
请提供一下代码～

**Comment by 13237143205** [comment]
已经解决了，用Texlive 2023模版编译不通过，目前尚不清楚原因，使用MikTex 就完全可以！



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Xiangdong Zeng ***@***.***&gt;
发送时间: 2024年2月25日 00:14
收件人: stone-zeng/fduthesis ***@***.***&gt;
抄送: 13237143205 ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [stone-zeng/fduthesis] 本地编辑不通过的问题，出现报错 (Discussion #322)





 
请提供一下代码～
